.github/workflows/ci-test-mac.yml

```diff
@@ -58,7 +58,7 @@ jobs:
       - name: Run tests (MacOS)
         run: |
           source ~/.bash_profile
-          ctest --output-on-failure --schedule-random -E "bpf|ubpf|testgen|smith"
+          ctest --output-on-failure --schedule-random -E "bpf|ubpf|testgen|smith|p4tc"
```

Comment:
I think it is best to disable these tests when some conditions are not met (eg CMake detects some runner is missing or the OS is not supported). But we can do this in a separate PR.